src/main/java/core/basesyntax/PrintInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface PrintInfo {
```

Comment:
better name suggestion
```suggestion
public interface Drawable {
```